lib/streamlit/config.py

```diff
@@ -589,6 +589,17 @@ def _server_port() -> int:
     return 8501
 
 
+_create_option(
```

Comment:
We may want to consider making this a hidden config option if our only anticipated use-case is by S4T.

It's likely that the product team will have an opinion on this, but my hunch is that we won't want to expose this config option to most users when running `streamlit config show` since it will probably be quite rarely used.